source/accounts/index.rst

```diff
@@ -4,10 +4,25 @@
 Accounts
 ########
 
-.. note::
+Getting an Internal RDHPCS Account
+==================================
+
```

Comment:
The breakdown of items below are good.  I think this section needs an "introduction".  I haven't looked below to see if something further down in the page would fit as an introduction here.

I feel that "Internal" should be removed from the header. We don't have an "external" RDHPCS account.

Reply:
Actually we use the "external" language to distinguish MSU. I have no objection to removing it, just explaining.